config.pri

```diff
@@ -8,17 +8,39 @@ unix:QMAKE_CXXFLAGS_RELEASE += -DNDEBUG
 win32:QMAKE_CXXFLAGS_RELEASE += /DNDEBUG
 
 # check Qt version
-QT_VERSION = $$[QT_VERSION]
-QT_VERSION = $$split(QT_VERSION, ".")
-QT_VER_MAJ = $$member(QT_VERSION, 0)
-QT_VER_MIN = $$member(QT_VERSION, 1)
-
-lessThan(QT_VER_MAJ, 5) {
-error(YACReader requires Qt 5 or newer but Qt $$[QT_VERSION] was detected.)
+defineTest(minQtVersion) {
+  maj = $$1
+  min = $$2
```

Comment:
I was about to suggest renaming these variables to clearer `major` and `minor` until I learned that this function definition is copied from *qtcreator.pri*. Now I agree that the names should stay abbreviated to facilitate updating from upstream.

Reply:
My hope is that we will have finished #73 before having to update this